CHANGELOG.md

```diff
@@ -14,7 +14,14 @@ All notable changes to this project are documented in this file.
 - Implement `ParametersHandler` library (https://github.com/dic-iit/bipedal-locomotion-controllers/pull/13)
 - Implement `GenericContainer::Vector` (https://github.com/dic-iit/bipedal-locomotion-controllers/pull/29)
 - Implement `Estimators` library (https://github.com/dic-iit/bipedal-locomotion-controllers/pull/23)
-- Renamed from ``bipedal-locomotion-controllers`` to ``bipedal-locomotion-framework`` (https://github.com/dic-iit/bipedal-locomotion-framework/pull/40).
+- Rename from `bipedal-locomotion-controllers` to `bipedal-locomotion-framework` (https://github.com/dic-iit/bipedal-locomotion-framework/pull/40).
```

Comment:
Not sure it make sense to have this line. The CHANGELOG is meant to track changes from one release to another, as this is the first changelog of an unreleased library, probably we can omit this.

Reply:
Ok So I can remove that line

Reply:
Done in 6f43ec8776e5ea89c460d471bc0410852bc6c9f5